apps/common-app/src/examples/LayoutAnimations/LayoutTransitionExample.tsx

```diff
@@ -0,0 +1,295 @@
+            transition: FadingTransition,
+          })
+        }
+      />
+      <Button
+        title="Jumping Transition"
+        onPress={() =>
+          navigation.navigate('jumping', {
+            title: 'JumpingTransition',
+            transition: JumpingTransition,
+          })
+        }
+      />
+      <Button
+        title="Curved Transition"
+        onPress={() =>
+          navigation.navigate('curved', {
+            title: 'CurvedTransition',
+            transition: CurvedTransition,
+          })
+        }
+      />
+      <Button
+        title="Entry/Exit Transition"
+        onPress={() =>
+          navigation.navigate('entryexit', {
+            title: 'EntryExit',
+            transition:
+              EntryExitTransition.entering(FlipInEasyX).exiting(FlipOutYRight),
+          })
+        }
+      />
+    </View>
+  );
+}
+
+const Stack = createStackNavigator<RootStackParamList>();
+
+export default function Layout() {
+  return (
+    <NavigationContainer independent={true}>
+      <Stack.Navigator screenOptions={{ title: 'Layout Transitions 🔥' }}>
+        <Stack.Screen name="Home" component={HomeScreen} />
+        <Stack.Screen
+          name="linear"
+          component={Transition}
+          initialParams={{
+            title: 'LinearTransition',
+            transition: LinearTransition,
+          }}
+        />
+        <Stack.Screen
+          name="fading"
+          component={Transition}
+          initialParams={{
+            title: 'FadingTransition',
+            transition: FadingTransition,
+          }}
+        />
+        <Stack.Screen
+          name="sequenced"
+          component={Transition}
+          initialParams={{
+            title: 'SequencedTransition',
+            transition: SequencedTransition,
+          }}
+        />
+        <Stack.Screen
+          name="jumping"
+          component={Transition}
+          initialParams={{
+            title: 'JumpingTransition',
+            transition: JumpingTransition,
+          }}
+        />
+        <Stack.Screen
+          name="curved"
+          component={Transition}
+          initialParams={{
+            title: 'CurvedTransition',
+            transition: CurvedTransition,
+          }}
+        />
+        <Stack.Screen
+          name="entryexit"
+          component={Transition}
+          initialParams={{
+            title: 'EntryExit',
+            transition:
+              EntryExitTransition.entering(FlipInEasyX).exiting(FlipOutYRight),
+          }}
+        />
+      </Stack.Navigator>
+    </NavigationContainer>
+  );
+}
+
+type TransitionProps = StackScreenProps<
+  RootStackParamList,
+  'linear' | 'fading' | 'sequenced' | 'jumping' | 'curved' | 'entryexit'
+>;
+
+function Transition({ route }: TransitionProps) {
+  const { title, transition } = route.params;
+  const [items, setItems] = useState(INITIAL_LIST);
+  const [selected, _] = useState({ label: title, value: transition });
```

Comment:
Then we don't need setter, so it seems like we can just write:
```suggestion
  const selected = { label: title, value: transition };
```